src/send_push/index.ts

```diff
@@ -0,0 +1,90 @@
+import https from 'node:https';
+import { Command, Option } from 'commander';
+import { createLogger } from './logger';
+
+export const sendPushCommand = new Command('send-push')
+  .description('Send a rich push notification to a specified device.')
+  .requiredOption(
+    '--api-key <apiKey>',
+    'App API key for your customer.io workspace. You can visit https://fly.customer.io/settings/api_credentials?keyType=app to get your API key.'
+  )
+  .requiredOption('--token <token>', 'Device token to send push to')
+  .addOption(
+    new Option('--platform <platform>', 'Platform of the device')
+      .choices(['ios', 'android'])
+      .makeOptionMandatory()
+  )
+  .addOption(
+    new Option(
+      '--provider <provider>',
+      'Push provider to use to deliver the notification'
+    ).choices(['apns', 'fcm'])
+  )
+  .option(
+    '--deep-link <deepLink>',
+    'Deep link to send in the test push notification'
+  )
+  .action(send);
+
+type sendOptions = {
+  apiKey: string;
+  token: string;
+  platform: string;
+  provider: string;
+  deepLink: string;
+};
+
+async function send(options: sendOptions) {
+  const logger = createLogger();
+
+  const data = JSON.stringify({
+    device_token: options.token,
+    device_platform: options.platform,
+    deep_link: options.deepLink,
+    push_provider: options.provider,
+  });
+
+  const reqOpts: https.RequestOptions = {
+    port: 443,
+    hostname: 'api.customer.io',
+    path: '/experimental/verify/push',
```

Comment:
Are there risks in this tool using an experimental path endpoint? Will this result in our tool being broken in the future if this path turns into a 404?

Reply:
good catch had meant to update that